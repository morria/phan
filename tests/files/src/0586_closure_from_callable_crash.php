<?php
class misc {
    protected function _sort() {
        $cb = Closure::fromCallable([$this, "foo"]);
        $cb('arg', []);
        $cb('arg', 'invalid');
    }

    protected function foo($type, array $item) {
        var_export([$type, $item]);
    }
}
