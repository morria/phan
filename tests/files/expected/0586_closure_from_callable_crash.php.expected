%s:6 PhanTypeMismatchArgument Argument 2 (item) is 'invalid' but \misc::foo() takes array defined at %s:9
